core/tests/test_utils.py

```diff
@@ -341,25 +341,26 @@ def swap_is_feature_flag_enabled_function(
     Yields:
         context. The context with function replaced.
     """
+    # Here we use unused-argument because as we are mocking a function,
+    # the args names should be similar to our mocking function args.
```

Comment:
Is this actually true? I think the previous version was better.

Also generally optional arguments are better placed at the end of the list of arguments.

Reply:
I have reverted the changes.